### PR TITLE
PADV-1113 add action column in courses table

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-1113

### Description
Add action column in courses table

### Changes made
Add action column for open course in LMS and add class in courses table
Update test

### Screenshots
Screenshot from 2024-04-03 09-31-01

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1980/courses

### CODE REVIEW

File: src/features/Courses/CoursesTable/columns.jsx [hunk 1]
```diff
@@ -30,6 +37,45 @@ const columns = [
     Header: 'Students invited',
     accessor: 'numberOfPendingStudents',
   },
+  {
+    Header: '',
+    accessor: 'className',
+    cellClassName: 'dropdownColumn',
+    Cell: ({ row }) => {
```
Comment (01001110J) [inline src/features/Courses/CoursesTable/columns.jsx:44]: This header shouldn't have this arrows, I think with a little css could be hidden ( not sure if this package has an api do this ) 

e

File: src/features/Courses/CoursesTable/columns.jsx [hunk 2]
```diff
@@ -30,6 +37,45 @@ const columns = [
...
+      const [isOpenModal, openModal, closeModal] = useToggle(false);
+      const courseDetailsLink = `${getConfig().LEARNING_MICROFRONTEND_URL}/course/${row.original.masterCourseId}/home`;
+      return (
+        <Dropdown className="dropdowntpz">
```
Comment (01001110J) [inline src/features/Courses/CoursesTable/columns.jsx:48]: This happens to me when I only have 2 courses, the modal is being cover, maybe a little bit more `z-index` is necessary. 

ee

File: src/features/Courses/CoursesTable/columns.jsx [hunk 3]
```diff
@@ -30,6 +37,45 @@ const columns = [
...
+          <Dropdown.Toggle
+            id="dropdown-toggle-with-iconbutton"
+            as={IconButton}
+            src={MoreHoriz}
+            iconAs={Icon}
+            variant="primary"
+          />
+          <Dropdown.Menu>
+            <Dropdown.Item href={courseDetailsLink} target="_blank">
```
Comment (01001110J) [inline src/features/Courses/CoursesTable/columns.jsx:57]: Missing [noopener](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) and [noreferrer](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer)

```suggestion
            <Dropdown.Item href={courseDetailsLink} target="_blank" rel="noopener noreferrer">**
```

File: src/features/Courses/CoursesTable/_test_/columns.test.jsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import { columns } from 'features/Courses/CoursesTable/columns';
 describe('columns', () => {
   test('returns an array of columns with correct properties', () => {
     expect(columns).toBeInstanceOf(Array);
-    expect(columns).toHaveLength(5);
+    expect(columns).toHaveLength(6);
```
Comment (01001110J) [inline src/features/Courses/CoursesTable/_test_/columns.test.jsx:6]: I would suggest to add a new test case to ensure the open can be open/closed and  if the actions are called.

**Review by 01001110J — CHANGES_REQUESTED** [review]

**Review by anfbermudezme — APPROVED** [review]

**Review by 01001110J — APPROVED** [review]

**Pull request merged**